atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -174,12 +175,13 @@ boolean allowHiddenTableAccess() {
      * successfully initialized. The TransactionManager stay uninitialized and continue to throw for all other purposes
      * until the callback returns at which point it will become initialized.
      *
-     * Note that if the callback throws an exception or blocks forever, the TransactionManager will never become
-     * initialized.
+     * Note that if the callback blocks forever, the TransactionManager will never become initialized. If the callback
+     * throws, the TransactionManager will not become initialized and it's close() method will be called. The callback
```

Comment:
it's --> its

*glares at the inconsistent grammatical rules English has*

The "it is" later in the sentence can become "it's" :joy: (I wouldn't do this, but really don't mind if you do)

Reply:
That's what I get for laughing at others